gulpfile.js

```diff
@@ -43,8 +43,13 @@ gulp.task("build", function () {
         .pipe(sourcemaps.write())
         .pipe(gulp.dest("./dist/node/"));
 
-    browserify({ entries: [ "./src/kuromoji.js" ] })
-        .bundle()
+    var b = browserify({
+        entries: ["./src/kuromoji.js"],
+        standalone: "kuromoji" // window.kuromoji
+    });
+    // replace NodeDictionaryLoader to BrowserDictionaryLoader
+    b.require(__dirname + "/src/loader/BrowserDictionaryLoader.js", {expose: "./loader/NodeDictionaryLoader.js"});
```

Comment:
This simulate [browser field](https://github.com/substack/node-browserify#browser-field "browser field").
This line meand contain  "BrowserDictionaryLoader.js" instead of "NodeDictionaryLoader.js".